.gitignore

```diff
@@ -1 +1,3 @@
-build/
\ No newline at end of file
+build/
+.vscode/
+.github/
```

Comment:
.github has some action config and can not be put inside .gitignore. 
Why do you need to add your gitignore to the PR?

Reply:
I will remove .vscode and .github but at the time of pushing code my build folder is also pushing to remove my build folder I put it in .gitignore.

Reply:
How do you add files to commit? Do you do git add . or git commit -m, is yes, then try only adding the files that you want to be present in the commit.
For example, if you only want files inside lib and include to be in the commit then do git add lib/ include/

Reply:
I have removed .gitignore